src/Core/UbuntuDriversBackend.vala

```diff
@@ -130,14 +125,28 @@ public class AppCenterCore.UbuntuDriversBackend : Backend, Object {
     }
 
     public async bool refresh_cache (Cancellable? cancellable) throws GLib.Error {
```

Comment:
When we refresh the caches of the other backends, also run the `ubuntu-drivers` command to detect any hardware changes or new drivers and store the result in GSettings.